allensdk/test/brain_observatory/behavior/test_write_nwb_behavior_ophys.py

```diff
@@ -9,37 +9,25 @@
 import allensdk.brain_observatory.nwb as nwb
 from allensdk.brain_observatory.behavior.session_apis.data_io import (
     BehaviorOphysNwbApi)
-from allensdk.brain_observatory.behavior.schemas import (
-    BehaviorTaskParametersSchema, OphysBehaviorMetadataSchema)
-
-
-@pytest.mark.parametrize('roundtrip', [True, False])
-def test_add_running_speed_to_nwbfile(nwbfile, running_speed, roundtrip, roundtripper):
```

Comment:
I think we still want to keep this test. Since the method it tests `get_running_speed()` is different from `get_running_data_df()`. I think we may even want to add a test for `get_raw_running_speed()` as well...

Reply:
It does the same thing as test_add_running_data_df_to_nwbfile since both check that the expected dataframe is the same as the obtained dataframe.